modules/openapi-generator/src/main/resources/typescript-fetch/runtime.mustache

```diff
@@ -102,7 +102,7 @@ export class BaseAPI {
 };
 
 export class RequiredError extends Error {
-    name: "RequiredError" = "RequiredError";
+    override name: "RequiredError" = "RequiredError";
```

Comment:
This change requires TS 4.3+, but the generated code should have compatibility with 2.4/3.9:

https://github.com/OpenAPITools/openapi-generator/blob/6392cc1876f54cc5bae1236586ae7f5630059bb7/modules/openapi-generator/src/main/resources/typescript-fetch/package.mustache#L26